packages/ui/Resizable/useResizable.js

```diff
@@ -0,0 +1,50 @@
+import {useCallback, useContext, useEffect, useRef} from 'react';
+
+import ResizeContext from './ResizeContext';
+
+/**
+ * A custom hook which returns handler that indicates a resize is necessary
+ * @function useResizable
+ * @memberof ui/Resizable
+ * @ui
+ */
+const useResizable = (props, config) => {
```

Comment:
Let's rename this component to `useResize`.

Reply:
I renamed to useResize